modules/ROOT/pages/osv2-faq.adoc

```diff
@@ -15,7 +15,9 @@ Also London (eu-west-2) can be accessed from the US cloud, but not the EU cloud.
 
 == What Mule versions support OSv2?
 
-Mule 3.8.5 and above, and all 4.x versions.
+Mule 3.8.5 and above, and all 4.x versions. 
+
+NOTE: Mule 4 only supports Object Store v2. Recent Mule 3 versions can work with either OSv2 or OSv1.
```

Comment:
Let's do something like this:
NOTE: Only Mule runtimes deployed to Cloudhub can use OSv2

Reply:
Fixed as requested.